src/validations.js

```diff
@@ -0,0 +1,12 @@
+export function validateEmail(email) {
+  if (!email) {
+    return 'validation_messages.error_required_field';
+  } else if (!/^([a-z\d[\]])([\w\-.+\][]*)@([ñ\w.\-\][]+)\.[\w\-\][.]{2,}(\?)?.*$/i.test(email)) {
+    return 'validation_messages.error_invalid_email_address';
+  }
+  return null;
+}
+
+export function validateRequiredField(field) {
```

Comment:
What do you think about calling it `value`?

```suggestion
export function validateRequiredField(value) {
```

Reply:
yes, it's more descriptive